src/thunderbots/CMakeLists.txt

```diff
@@ -107,6 +108,24 @@ add_executable (ai_logic
 add_dependencies(ai_logic ${catkin_EXPORTED_TARGETS})
 target_link_libraries(ai_logic ${catkin_LIBRARIES})
 
+add_executable (backend_output
+#            ${PROTO_SRCS}
+            backend_output/main.cpp
+            ai/primitive/primitive.h
+            ai/primitive/move_prim.h
+            ai/primitive/move_prim.cpp
+            backend_output/backend.cpp
+            backend_output/backend.h
+            backend_output/grsim/grsim_backend.cpp
+            backend_output/grsim/grsim_backend.h
+
```

Comment:
Are we not going to "GLOBing" files? Might be best to get this setup now before the number of files we have blows up.

Reply:
Will change back to globbing :+1: